eras/conway/impl/src/Cardano/Ledger/Conway/Delegation.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE TypeFamilies #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Cardano.Ledger.Allegra.Delegation () where
+
+import Cardano.Ledger.Allegra.Era
+import Cardano.Ledger.Crypto
+import Cardano.Ledger.Shelley.Delegation
+
+instance Crypto c => EraDCert (AllegraEra c) where
```

Comment:
I think we may have missed out renaming `Allegra` to `Conway` here.

Reply:
good catch, looks like this file isn't even referenced in the cabal file